src/Augurk/Content/augurk-additions.css

```diff
@@ -14,6 +14,14 @@
  limitations under the License.
 */
 
+html, body {
+    height: 100%;  
+}
+
+.container {
+    height: calc(100% - 75px);
```

Comment:
Maybe leave a comment here why the 75px?

Reply:
Sure